tensorflow_addons/optimizers/average_wrapper.py

```diff
@@ -25,19 +25,29 @@
 class AveragedOptimizerWrapper(KerasLegacyOptimizer, metaclass=abc.ABCMeta):
     @typechecked
     def __init__(
-        self, optimizer: types.Optimizer, name: str = "AverageOptimizer", **kwargs
+        self,
+        optimizer: types.Optimizer,
+        name: str = "AverageOptimizer",
+        **kwargs,
     ):
         super().__init__(name, **kwargs)
 
         if isinstance(optimizer, str):
-            optimizer = tf.keras.optimizers.get(optimizer)
+            if (
```

Comment:
Do you think that we could call an external function with this logic instead of replicating the code in every wrapper?